website/docs/cookbooks/gh-action.md

```diff
@@ -0,0 +1,162 @@
+
+`scala-cli` lets you run, test, and package Scala code in various environments, also in GitHub CI. 
+To use Scala CLI features in a simple way you can use the GitHub Actions [scala-cli-setup](https://github.com/VirtusLab/scala-cli-setup) that installs everything necessary to run the Scala application and more.
+
+For example, here's a simple `ls` application to printing the files in the given directory:
+```scala title=Ls.scala
+//> using scala "2.13"
+//> using lib "com.lihaoyi::os-lib:0.7.8"
+
+@main def hello(args: String*) =
+  val path = args.headOption match
+    case Some(p) => os.Path(p, os.pwd)
+    case _       => os.pwd
+
+  if (os.isDir(path)) println(os.list(path).mkString(","))
+  else System.err.println("Expected directory path as a input")
+```
+
+and some tests for `ls` application:
+
+```scala title=TestsLs.test.scala
+//> using lib "org.scalameta::munit::0.7.27"
+import scala.util.Properties
+
+class TestsLs extends munit.FunSuite {
+  test("ls") {
+    // prepare test directory
+    val tempDir = os.temp.dir()
+    // create files
+    val exptectedFiles = Seq("Ls", "Hello").map(tempDir / _)
+    exptectedFiles.foreach(os.write(_, "Hello"))
+
+    // check
+    val scalaCLILauncher = if(Properties.isWin) "scala-cli.bat" else "scala-cli"
+    val foundFiles =
+      os.proc(scalaCLILauncher, "Ls.scala", "--", tempDir).call().out.text().trim
+
+    exptectedFiles.map(_.toString).foreach { file =>
+      assert(foundFiles.contains(file))
+    }
+  }
+}
+
+```
+
+## Run tests in Github CI
+
+The following configuration of `ci.yml` contains a definition of job that run tests using `scala-cli` for every platform defined in `matrix.OS`.
+
+```yaml
+jobs:
+  build:
+    runs-on: ${{ matrix.OS }}
+    strategy:
+      matrix:
+        OS: ["ubuntu-latest", "macos-latest", "windows-latest"]
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+        submodules: true
+    - uses: coursier/cache-action@v6.3
+    - uses: VirtusLab/scala-cli-setup@v0.1
+    - run: scala-cli test .
+```
+
+## Test your scala code style
+
+To test you code style of your application you can use [Scalafmt](https://scalameta.org/scalafmt/). 
+
+Before running using `Scalafmt` there is needed to create the `.scalafmt.conf` configuration file:
+
+```scala
+version = 3.5.2
+runner.dialect = scala3
+```
+
+and then test your code style in GitHub CI by adding new job `check`:
+```yaml
+  check:
+    runs-on: "ubuntu-latest"
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+        submodules: true
+    - uses: coursier/cache-action@v6.3
+    - uses: VirtusLab/scala-cli-setup@v0.1
+    - name: Scalafmt check
+      run: |
+        scala-cli fmt --check . || (
+          echo "To format code run"
+          echo "  scala-cli fmt ."
+          exit 1
+        )
+```
+
+If `scala-cli fmt --check .` command fails, it can be easily fixed by running `scala-cli fmt .` which correctly formats your code.
```

Comment:
```suggestion
If the `scala-cli fmt --check .` command fails, it can be easily fixed by running `scala-cli fmt .`, which correctly formats your code.
```